app/services/email_template_parser.rb

```diff
@@ -0,0 +1,76 @@
+class EmailTemplateParser
+  def initialize(conference, user)
+    @conference = conference
+    @user = user
+  end
+
+  # rubocop:disable Metrics/AbcSize, Metrics/ParameterLists
+  def retrieve_values(event = nil, booth = nil, quantity = nil, ticket = nil)
+    h = {
+      'email'                  => @user.email,
+      'name'                   => @user.name,
+      'conference'             => @conference.title,
+      'conference_start_date'  => @conference.start_date,
+      'conference_end_date'    => @conference.end_date,
+      'registrationlink'       => Rails.application.routes.url_helpers.conference_conference_registration_url(
```

Comment:
The default here should not be local host, there should be a way to get the current hostname

Reply:
Also, I think `registrationlink` should really be `registration_link`